server/opendp_apps/analysis/testing/test_dp_histogram_spec.py

```diff
@@ -0,0 +1,352 @@
+from os.path import abspath, dirname, isdir, isfile, join
+
+CURRENT_DIR = dirname(abspath(__file__))
+TEST_DATA_DIR = join(dirname(dirname(dirname(CURRENT_DIR))), 'test_data')
+
+import json
+import decimal
+
+from django.contrib.auth import get_user_model
+from django.test.testcases import TestCase
+
+from opendp_apps.analysis.analysis_plan_util import AnalysisPlanUtil
+from opendp_apps.analysis.models import AnalysisPlan
+from opendp_apps.dataset.models import DataSetInfo
+
+from unittest import skip
+from opendp_apps.analysis.tools.dp_histogram_spec import DPHistogramSpec
+
+from opendp_apps.model_helpers.msg_util import msgt
+from opendp_apps.utils.extra_validators import *
+
+
+class HistogramStatSpecTest(TestCase):
+
+    fixtures = ['test_dataset_data_001.json', ]
+
+    def setUp(self):
+        """Make a user"""
+        self.user_obj, _created = get_user_model().objects.get_or_create(username='dev_admin')
+
+    def retrieve_new_plan(self):
+        """Convenience method to create a new plan"""
+
+        # Create a plan
+        #
+        dataset_info = DataSetInfo.objects.get(id=4)
+
+        plan_info = AnalysisPlanUtil.create_plan(dataset_info.object_id, self.user_obj)
+        self.assertTrue(plan_info.success)
+        orig_plan = plan_info.data
+
+        # Retrieve it
+        #
+        analysis_plan = AnalysisPlan.objects.first()
+        self.assertEqual(orig_plan.object_id, analysis_plan.object_id)
+
+        return analysis_plan
+
+    def test_10_debug(self):
```

Comment:
Add a similar test using:
```json
{"variable": "Subject",
 "col_index": 0,
 "fixed_value": "ac",
 "variable_info": {"categories': ["mh", "ac", "jm", "cw", "bh2"],
                             "type": "Categorical"},
  "fixed_value": "ac"
  }                         
```
Note: I'm pull info from: https://github.com/opendp/dpcreator/blob/develop/server/test_data/non_dp_debug_info/info_Fatigue_data.json#L21
- ^ includes more categories under https://github.com/opendp/dpcreator/blob/f2479e7b86fd0d08542acdc90add117ef2419d8a/server/test_data/non_dp_debug_info/info_Fatigue_data.json#L55

Reply:
Similar for actually running data through...